mmv1/templates/terraform/examples/dataplex_datascan_primary.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_dataplex_datascan" "<%= ctx[:primary_resource_id] %>" {
+  location     = "us-central1"
+  name         = "tf-test-datascan%{random_suffix}"
+  description  = "Test Datascan"
+  display_name = "Test Datascan"
+
+  data = {
+	entity = "projects/<%= ctx[:test_env_vars]['project_name'] %>/locations/us-central1/lakes/amandeep-dev-lake/zones/rawzone/entities/austin_311"
```

Comment:
should we not hardcode these values?

Reply:
just to make sure I understand, we have to create a datascan against a source

are you suggesting we create a new data source (entity) and then reference that as the entity in the datascan
(or possibly reference a public entity)?


---

Here, it looks like there's a hard-coded entity, which the tests wont have reference to during automated tests, and hence will fail?